File: app/frontend/ea-project/src/models/output/UploadUserPhotoOutput.js
import BaseOutput from "./BaseOutput";

export default class UploadUserPhotoOutput extends BaseOutput {
  constructor(token = '', updateSuccessful = false, feedbackMessages = []) {
    super(feedbackMessages);
    this.token = token;
    this.updateSuccessful = updateSuccessful;
  }
}

File: app/backend/src/main/java/eaproject/output/UploadUserPhotoOutput.java
package eaproject.output;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public class UploadUserPhotoOutput extends BaseOutput implements Serializable {

    @JsonProperty
    private String token;

    @JsonProperty
    private boolean updateSuccessful;

    public boolean isUpdateSuccessful() {
        return updateSuccessful;
    }

    public void setUpdateSuccessful(boolean updateSuccessful) {
        this.updateSuccessful = updateSuccessful;
    }

    public String getToken() { return token; }

    public void setToken(String token) { this.token = token; }
}
